README.md

```diff
@@ -7,6 +7,8 @@ UBC Snowbots Repository for competitions.
 - [Installation and Setup](#installation-and-setup)
   - [Important Notes](#important-notes)
   - [Zed Configuration](#zed-configuration)
+  - [Rosinstall](#external-packages-with-rosinstall)
```

Comment:
Ah, ok, in that case could we rename this to `External Packages` and make a note about running `source external_pkgs/setup.sh` before being able to use external packages?

Reply:
Changed this